geonode/geonode/catalogue/templates/catalogue/full_metadata.xml

```diff
@@ -413,10 +414,10 @@
            <gmd:temporalElement>
              <gmd:EX_TemporalExtent>
                <gmd:extent>
-                 <gml:TimePeriod gml:id="T_01">
-                   <gml:beginPosition>{{layer.temporal_extent_start|date:"c"}}</gml:beginPosition>
-                   <gml:endPosition>{{layer.temporal_extent_end|date:"c"}}</gml:endPosition>
-                 </gml:TimePeriod>
+                 <gml32:TimePeriod gml32:id="T_01">
```

Comment:
@ahmdthr why not just remove the old gml namespace completly. As far I can see, no gml prior to 3.2 is being used (anymore). Then you'd keep the `gml` prefix.

Reply:
pycsw will add `xmlns:gml="http://www.opengis.net/gml"` for ISO, so we will have to create a PR to pycsw repo to not include it.

Reply:
This [code fragment](https://github.com/geopython/pycsw/blob/c532be3aba731bfee093620c40f64160c3901f8e/pycsw/ogc/csw/csw2.py#L1042) adds all namespaces from the pycsw configuration [default namespaces](https://github.com/geopython/pycsw/blob/c532be3aba731bfee093620c40f64160c3901f8e/pycsw/core/config.py#L72).

In my opinion, we have two options:

1- Use this PR as it is, it is GDI-DE compliant.
2- Create a PR on pycsw to remove gml from `namespaces` or `keep_ns_prefixes` from the [`config.py`](https://github.com/geopython/pycsw/blob/c532be3aba731bfee093620c40f64160c3901f8e/pycsw/core/config.py) file. This will allow us to replace 

`<gmd:MD_Metadata xmlns:gmd="http://www.isotc211.org/2005/gmd" xmlns:gml="http://www.opengis.net/gml" xmlns:gmx="http://www.isotc211.org/2005/gmx" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:gco="http://www.isotc211.org/2005/gco" xsi:schemaLocation="http://www.isotc211.org/2005/gmd http://www.isotc211.org/2005/gmd/gmd.xsd">
`
with 

`<gmd:MD_Metadata xmlns:gmd="http://www.isotc211.org/2005/gmd" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:gmx="http://www.isotc211.org/2005/gmx" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:gco="http://www.isotc211.org/2005/gco" xsi:schemaLocation="http://www.isotc211.org/2005/gmd http://www.isotc211.org/2005/gmd/gmd.xsd">`

on `full_metadata.xml`. This makes us avoid using gml32 for `<gmd:EX_TemporalExtent>`.


What do you think about this?

Reply:
If it validates I am fine with it for now.